collect/cache/cache.go

```diff
@@ -18,11 +18,18 @@ type Cache interface {
 	// GetAll is used during shutdown to get all in-flight traces to flush them
 	GetAll() []*types.Trace
 
+	GetCacheSize() int
```

Comment:
I think this shoulid be renamed to `GetCacheCapacity` and include a comment on what units it's in (traces, bytes, spans, etc).